lib/node.js

```diff
@@ -962,17 +962,22 @@ declare class stream$Writable extends stream$Stream {
   constructor(options?: writableStreamOptions): void;
   cork(): void;
   end(
-    chunkOrEncodingOrCallback?: string | Buffer | Function,
+    chunkOrEncodingOrCallback?: Buffer | string | Function,
     encodingOrCallback?: string | Function,
     callback?: Function
   ): void;
   setDefaultEncoding(encoding: string): boolean;
   uncork() : void;
   write(
-    chunk: string | Buffer,
+    chunk: Buffer | string,
     encodingOrCallback?: string | Function,
     callback?: Function
   ): boolean;
+  _write(
+    chunk: Buffer | string,
+    encoding: string,
+    callback: (error: ?Error, data?: Buffer | string) => void
+  ): boolean;
```

Comment:
This is part of Writable: https://nodejs.org/dist/latest-v5.x/docs/api/stream.html#stream_writable_write_chunk_encoding_callback_1